starboard/shared/win32/posix_emu/include/netinet/in.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef STARBOARD_SHARED_WIN32_POSIX_EMU_INCLUDE_NETINET_IN_H_
+#define STARBOARD_SHARED_WIN32_POSIX_EMU_INCLUDE_NETINET_IN_H_
+
+#include <arpa/inet.h>
```

Comment:
Does this imply that partners' toolchains must ensure including `netinet/in.h` makes htonl() and friends available?

Reply:
I want to make sure I understand the wording on the POSIX documentation for the header—does it require that i.e. `htonl()` be available when `netinet/in.h` (and not `arpa/inet.h`) is included, or is it saying it `may` be made available when only `netinet/in.h` is available?